server/test/categoryRouter.test.js

```diff
@@ -146,9 +178,46 @@ describe('categoryRouter', () => {
         });
     });
   });
+  
+  describe('/getCategoryItems/:categoryId', () => {
+    it('get items in testCategory1, should return response.body.items with 2 items', (done) => {
+      chai
+        .request(server)
+        .get('/category/getCategoryItems/' + testCategory1._id)
+        .end((error, response) => {
+          expect(response).to.have.status(200);
+          expect(response.body.title).equals('User1 Category1');
+          expect(response.body.items.length).equals(2);
```

Comment:
Use expect('foo').to.have.lengthOf(3) format

Reply:
noted!